### DISCUSSION TITLE
How to join the Elsa community?

### DISCUSSION THREAD

**Discussion by shxylgithub** [elsa-workflows/elsa-core#5638] [Q&A]
How to join the Elsa community?

**Comment by glime-ai** [comment]
You can join the Elsa community by visiting the following platforms:

1. GitHub: Participate in discussions, report issues, and contribute to the codebase on the [Elsa Workflow GitHub repository](https://github.com/elsa-workflows/elsa-core).
2. Discord: Join the Elsa community on Discord for real-time chat and support at [https://discord.com/invite/hhChk5H472](https://discord.com/invite/hhChk5H472).
3. Stack Overflow: Ask questions and find answers on Stack Overflow using the tag [elsa-workflows](https://stackoverflow.com/questions/tagged/elsa-workflows).

These platforms are great resources for getting help, sharing your expertise, and staying up-to-date with the latest developments in the Elsa Workflow project.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
